src/transformers/tokenization_xlm_roberta.py

```diff
@@ -155,7 +163,7 @@ def __init__(
         # The first "real" token "," has position 4 in the original fairseq vocab and position 3 in the spm vocab
         self.fairseq_offset = 1
 
-        self.fairseq_tokens_to_ids["<mask>"] = len(self.sp_model) + len(self.fairseq_tokens_to_ids)
+        self.fairseq_tokens_to_ids["<mask>"] = len(self.sp_model) + self.fairseq_offset
```

Comment:
Closes #2741

Reply:
See comment about the mask token.

https://github.com/huggingface/transformers/pull/3198/files#r390010227